### PR TITLE
EDU-2367-EN-Create-API-overview

### PR DESCRIPTION
**Pull request by peeetry**
Related issue: [EDU-2367](https://aziontech.atlassian.net/browse/EDU-2367)

Changes:

* Introducing new file: `Azion API overview` for the documentation portal.

> This overview brings all necessary knowledge to make Azion RESTful API requests. For more information on Postman UI, and how to make Azion API request through Postman UI, reach out to the API oficial documentation. There you will find this same page, but with more content about Postman UI. Postman UI is out of here because is only relevant for Postman users, and the API overview is relevant for all API users.

[EDU-2367]: https://aziontech.atlassian.net/browse/EDU-2367?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title: Azion API
+description: Azion RESTful API. Learn more about oficial endpoints, methods and API use cases.
+meta_tags: azion, rest api, restful api, azion restful api
+namespace: documentation_products_azion_api_overview
+permalink: /documentation/products/azion-api-overview/
+---
+
+# Getting Started
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:9]:
```suggestion
```

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+permalink: /documentation/products/azion-api-overview/
+---
+
+# Getting Started 
+
+This documentation will guide you on the first steps of using the **Azion API**.
+
+## API overview
+
+The **Azion API** is a *RESTful API* based on *HTTPS* requests, that allows users to fully interact with Azion products through API requests. Most API endpoints have the main HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE`), so you can create, read, update, and delete Azion configuration instances. All changes made through the API are reflected on the [Real-Time Manager (RTM)](https://manager.azion.com) web platform.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:15]:
```suggestion
The **Azion API** is a *RESTful API* based on *HTTPS* requests, that allows users to fully interact with Azion products through API requests. Most API endpoints have the main HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE`), so you can create, read, update, and delete Azion configuration instances. All changes made through the API are reflected on the [Real-Time Manager (RTM)](https://manager.azion.com).
```

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+> Access the [introduction](https://api.azion.com/#intro) documentation to find reference information about the API.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:17]: component tip

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+## Prerequisites
+
+To run **Azion API** requests you need:
+
+- **Real-Time Manager (RTM)** account.
+- **API token (personal token)**.
+
+> Learn [how to create an RTM account](https://www.azion.com/en/documentation/products/accounts/creating-account/) and how to create a [personal token](https://www.azion.com/en/documentation/products/accounts/personal-tokens/) to use with the API.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:26]:
```suggestion
> Learn [how to create an RTM account](/en/documentation/products/accounts/creating-account/) and how to create a [personal token](/en/documentation/products/accounts/personal-tokens/) to use with the API.
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:26]: componente

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx (hunk 3 above)
Comment (peeetry) [reply src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:15]:
```suggestion
:::tip
> Access the [introduction](https://api.azion.com/#intro) documentation to find reference information about the API.
:::
```

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx (hunk 4 above)
Comment (peeetry) [reply src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:26]:
```suggestion
:::tip
> Learn [how to create an RTM account](https://www.azion.com/en/documentation/products/accounts/creating-account/) and how to create a [personal token](https://www.azion.com/en/documentation/products/accounts/personal-tokens/) to use with the API.
:::
```

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+meta_tags: azion, rest api, restful api, azion restful api
+namespace: documentation_products_azion_api_overview
+permalink: /documentation/products/azion-api-overview/
+---
+
+This documentation will guide you on the first steps of using the **Azion API**.
+
+## API overview
+
+The **Azion API** is a *RESTful API* based on *HTTPS* requests, that allows users to fully interact with Azion products through API requests. Most API endpoints have the main HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE`), so you can create, read, update, and delete Azion configuration instances. All changes made through the API are reflected on the [Real-Time Manager (RTM)](https://manager.azion.com).
```
Comment (hannah-sk) [inline src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:13]:
```suggestion
The **Azion API** is a *RESTful API* based on *HTTPS* requests, that allows users to fully interact with Azion products through API requests. Most API endpoints have the main HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE`), so you can create, read, update, and delete Azion configuration instances. All changes made through the API are reflected on [Real-Time Manager (RTM)](https://manager.azion.com).
```

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+:::tip
+> Access the [introduction](https://api.azion.com/#intro) documentation to find reference information about the API.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:16]:
```suggestion
Access the [API introduction](https://api.azion.com/#intro) documentation to find reference information about the API.
```

Comment (hannah-sk) [reply src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:16]: só p tentar ficar mais claro que vai sair dessa doc

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+:::
+
+## Prerequisites
+
+To run **Azion API** requests you need:
+
+- **Real-Time Manager (RTM)** account.
+- **API token (personal token)**.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:24]:
```suggestion
- A **Real-Time Manager (RTM)** account.
- An **API token (personal token)**.
```

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+:::tip
+> Learn [how to create an RTM account](https://www.azion.com/en/documentation/products/accounts/creating-account/) and how to create a [personal token](https://www.azion.com/en/documentation/products/accounts/personal-tokens/) to use with the API.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:27]:
```suggestion
See [how to create an RTM account](/en/documentation/products/accounts/creating-account/) and how to create a [personal token](/en/documentation/products/accounts/personal-tokens/) to use with the API.
```

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+:::
+
+## Make API requests with curl (and other programming languages) snippets
+ 
+Azion API documentation offers code examples to make requests with the terminal, using the curl tool, or to automate them with programming languages.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:32]:
```suggestion
Azion API documentation offers code examples to make requests with the terminal, using the `curl` command, or to automate them with several programming languages.
```

Comment (hannah-sk) [reply src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:32]: a doc oferece ou apenas o Postman, como diz no parágrafo abaixo?

File: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+ 
+Azion API documentation offers code examples to make requests with the terminal, using the curl tool, or to automate them with programming languages. 
+
+Postman offers HTTPS request snippets for many programming languages.
+
+To make Azion API requests with curl:
+
+1. Access the [Azion API documentation](https://api.azion.com).
+2. On the first column, select the endpoint and method you would like to perform. For example `Tools` > `Personal Tokens` > `POST Create Personal Token`.
+3. Navigate to `Example Request`.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx:40]:
```suggestion
3. Navigate to **Example Request.**
```

**Comment by peeetry** [comment]
Closed to work on article structure.

**Pull request closed without merging**